### ISSUE TITLE
Compact clocks in timing diagrams not rendered correctly

### ISSUE THREAD

**Issue by jonnykl** [plantuml/plantuml#1578]
**Describe the bug**
Compact clocks in timing diagrams are not rendered correctly. Tested using the PlantUML Web Server and locally using v1.2023.11. The bug occurs in latex, eps and png export mode. I did not test other modes.

**To Reproduce**

```plantuml
@startuml
mode compact
clock "clk" as clk with period 1
binary "bin" as bin
@0
bin is 0
@1
bin is 1
@enduml
```

**Expected behavior**
I would expect no lines below and to the right of "clk".

**Screenshots**
[Generated image of the code given above](https://www.plantuml.com/plantuml/png/DSt12GCn34JHkweONC6w0xMYo8Q8jLUBhH3IVKGWfpU7WSyxPCLh3fhUEjJdBHgam_L4qN4Mo4QAjyKJTr_c3PKUTidweAI_HqfyK0BRE8ZhVrVYVhKiV040)

**Comment by arnaudroques** [comment]
> I would expect no lines below and to the right of "clk".

Sorry, I don't understand this sentence.

![](https://www.plantuml.com/plantuml/png/DSt12GCn34JHkweONC6w0xMYo8Q8jLUBhH3IVKGWfpU7WSyxPCLh3fhUEjJdBHgam_L4qN4Mo4QAjyKJTr_c3PKUTidweAI_HqfyK0BRE8ZhVrVYVhKiV040)

Are we talking about this graphical glitch:  the dotted line that enters the frame with the wording "clk".

Thanks!

**Comment by jonnykl** [comment]
> Are we talking about this graphical glitch: the dotted line that enters the frame with the wording "clk".

Yes, I think there should be no frame.

**Comment by The-Lum** [comment]
Hello all, and PlantUML team,

Compare _(without `mode compact`)_:
```puml
@startuml
'mode compact

clock   "Clock_0"   as C0 with period 50
clock   "Clock_1"   as C1 with period 50 pulse 15 offset 10
binary  "Binary"  as B
concise "Concise" as C
robust  "Robust"  as R
analog  "Analog"  as A

@0
C is Idle
R is Idle
A is 0

@100
B is high
C is Waiting
R is Processing
A is 3

@300
R is Waiting
A is 1
@enduml

```
>[![](https://www.planttext.com/api/plantuml/svg/NP0z3y8W443t-nKN5bU8wTwFoSrqSJH8iILIhW4Qux-NZsYCsxkxznXeGvG-xgk5mqgJHaNh9bK4K9RK0n7PaE7AMM8PSE3uD771JNj34pRyNnGVKVo9kEqsQ1GDqlqUT4J1uMQSzA_KzWLOANjGv9H9BXigiF8WUBhj8IPxB53j4QIJbkQqxWhKTGS0BOS1JS3JP3MCN-eoyNGMd4EVXyNCIrKlqaJZvcgVFIaTGfvBSqpDCJNZhrak0bhjfloBRm00)](https://www.planttext.com/?text=NP0z3y8W443t-nKN5bU8wTwFoSrqSJH8iILIhW4Qux-NZsYCsxkxznXeGvG-xgk5mqgJHaNh9bK4K9RK0n7PaE7AMM8PSE3uD771JNj34pRyNnGVKVo9kEqsQ1GDqlqUT4J1uMQSzA_KzWLOANjGv9H9BXigiF8WUBhj8IPxB53j4QIJbkQqxWhKTGS0BOS1JS3JP3MCN-eoyNGMd4EVXyNCIrKlqaJZvcgVFIaTGfvBSqpDCJNZhrak0bhjfloBRm00)

with `mode compact`:
>[![](https://www.planttext.com/api/plantuml/svg/XPBFQiGW4CRlynI3VO0QmjwMaZ-z57eekVHON7KJMQD13MKf--ur9YbBbfBRD_FzFdK62n-O2sEleRT28hVzm7Y0emzNBLyWg5wPzbMnrh4UVm3nNHbPMsqTEYbY7GcfOo-Pz_PJ7XktXMYT4lyJN5j-UR05SvVMIMbMoXgk_ED5MyxPq-Z3BdPIXhdhBiOCqxRTmMvmV5usDi-2IEf9V54IDVDOK_nMeSD1EcK57kWMp5Omsu0uZ3fEdHtGdizU1imeB6z7KYL1KhA2TKEadWL91yAo42HD4ZFTm38PaZA9kLq2G46XHkNnJMW9pPyg9qMZdL4ArLHqgkrcz9Ef43_AJ7yuowNtKvqoUSpaCTFSayd9e916JB_l5m00)](https://www.planttext.com/?text=XPBFQiGW4CRlynI3VO0QmjwMaZ-z57eekVHON7KJMQD13MKf--ur9YbBbfBRD_FzFdK62n-O2sEleRT28hVzm7Y0emzNBLyWg5wPzbMnrh4UVm3nNHbPMsqTEYbY7GcfOo-Pz_PJ7XktXMYT4lyJN5j-UR05SvVMIMbMoXgk_ED5MyxPq-Z3BdPIXhdhBiOCqxRTmMvmV5usDi-2IEf9V54IDVDOK_nMeSD1EcK57kWMp5Omsu0uZ3fEdHtGdizU1imeB6z7KYL1KhA2TKEadWL91yAo42HD4ZFTm38PaZA9kLq2G46XHkNnJMW9pPyg9qMZdL4ArLHqgkrcz9Ef43_AJ7yuowNtKvqoUSpaCTFSayd9e916JB_l5m00)

--- 

Then the main goal (with `compact` mode) will be:
- that `compact` mode should suppress the red frames around clock name.

_If that can precise the issue._
Regards,
Th.

**Comment by The-Lum** [comment]
Hi all,

This is now fixed, on [v1.2024.8](https://github.com/plantuml/plantuml/releases/tag/v1.2024.8)
- Thanks @jonnykl for the report.
- Thanks PlantUML team for your work.

Regards.
